lib/glimesh/streams.ex

```diff
@@ -125,7 +125,13 @@ defmodule Glimesh.Streams do
   end
 
   def get_channel_for_user!(user) do
-    Repo.get_by(Channel, user_id: user.id) |> Repo.preload([:category, :user])
```

Comment:
Did we need the  |> Repo.preload([:category, :user]) for something?

Reply:
Good question, I'll add it back into the the function. Since it didn't break anything not being there but it could be an edge case that breaks from it.